src/element.js

```diff
@@ -54,6 +54,7 @@ function boundingRects(rect, dpr) {
     y: bounds.top,
     w: width,
     h: height,
+    active: rect.active,
```

Comment:
Fix looks wrong to me, the bounding rects are not elements and should not have colors etc. So something else is wrong.

Reply:
See https://github.com/kurkle/chartjs-chart-treemap/issues/149#issuecomment-1281269100

Reply:
If we don't want to add to the bounding rects, we need to pass it as argument to draw labels and captions functions.